package.json

```diff
@@ -21,7 +21,8 @@
     "proxy-agent": "^3.0.1",
     "puppeteer": "^2.0.0",
     "wdio-sauce-service": "^0.4.4",
-    "webdriverio": "^4.14.2"
+    "webdriverio": "^4.14.2",
+    "jwt-decode": "^3.0.0"
```

Comment:
How about removing it from devDependencies to avoid duplicates?

Reply:
i added it do `devDependencies` with this pr, i just want to test and see if it works if i put it in theire, and then remove it from `dependencies`

Reply:
removed the dupe from dependecies noq